tests/unit/phpDocumentor/UriTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * This file is part of phpDocumentor.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @copyright 2010-2015 Mike van Riel<mike@phpdoc.org>
+ * @license   http://www.opensource.org/licenses/mit-license.php MIT
+ * @link      http://phpdoc.org
+ */
+
+namespace phpDocumentor;
+
+/**
+ * Test case for Uri
+ *
+ * @coversDefaultClass phpDocumentor\Uri
+ */
+final class UriTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @covers ::__construct
+     * @covers ::validateString
+     * @covers ::validateUri
+     */
+    public function testItShouldBuildAUri()
+    {
+        $uri = new Uri('file://foo');
+
+        $this->assertInstanceOf(Uri::class, $uri);
```

Comment:
This check is redundant; there is no way that the $uri can be anything other than a class of type Uri